performant-software/textlab
# Fixes location of popover on click in leaf view

**What this Does**
Fixes the calculation for positioning the popover. Adds resize handling. 

**How to Test**
Go here: https://mel-textlab-pr-189.herokuapp.com/

Go to an editor window with a leaf and click on an annotation

Observe that the popover appears in the correct place. 

Please also test this at varying zoom levels and resize the browser window and the internal panes (slide the dividers around)
